fms/models/llama.py

```diff
@@ -246,6 +246,36 @@ def reset_parameters(self):
             ):
                 m.reset_parameters()
 
+    def validate_reset_parameters(self):
```

Comment:
we should have a comment explaining the purpose of this function, not obvious why it's needed but ensuring FSDP didn't break anything makes sense